packages/react-sdk/src/expression.ts

```diff
@@ -141,25 +145,37 @@ export const executeExpressions = (
     }
   );
 
-  // execute chain of expressions
```

Comment:
sortedExpressions  above seem like not have transitivity rule for sorting i.e.
if a<b and b<c then a<c
seems like doesn't work for algotithms above.
I think that transitivity required for most sorting algorithms. Not sure 100% but pretty sure that its something every fast algorithm will rely on.